notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>Some general comments. </p><p> </p><ul><li>In between I have access to a 4th Gen Machine I will add some general comments here. </li><li>Could you please add a description cell here. Include links or figures to help to describe the further actions. </li><li>In functions could you please add docstring format and type hints, that will help readers to understand better the code without the need to debug the code.</li><li>Could you also add some comments in the code. Not all people are familiar with HuggingFace and Optimum.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/774/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks @AlexKoff88 for your contribution, I have started the process with OneCloud (48 hours), but DevCloud doesn't give me the option to access the machine without payment. So, I am figuring out how to access the 4th Gen. I think that also we need to include a note in the notebook to show the reader how they could test this notebook for free. The Tiny Machine option in DevCloud could be a broader solution now, but I found a gap in launching that instance. 

For now I added some comments to your notebook. As soon I have access I will test the notebook. Thanks and BR.

Reply:
@AlexKoff88, trying to review your notebooks, I am facing some bottlenecks:

- Devcloud has a bug for free access, and it will be solved W3/4. (This will be the option for broader audiences)
- I contacted an internal team, and I could access that after Monday. 
- I am trying OneCloud, but I am facing some problems with the Git Proxy. And I am not sure if I could fix that on those machines.

I have added some comments on the notebook, but I think we need to add some notes about how users can access the processor.

I will jump into that once I have a solution. I am trying to be faster on this review, but I found a lot of bottlenecks.  :(

Best regards.

Reply:
Thanks @paularamo, I applied your comments related to the code. As for the device availability, I think this is a common problem and is not related directly to the 4th Gen Xeon, even though it is not widely spread. 
My assumption is that this tutorial is not for everybody for sure but it can be helpful for those who would like to get all the benefits of the new HW generation in case it is available.